hooks/post_gen_project.sh

```diff
@@ -17,7 +20,29 @@ git add .
 git commit -am "Setup Nesta Data Science cookiecutter" -q
 git checkout -b dev -q
 
-echo "Configured git repo at $(pwd), please enter the project directory & run 'make install'"
+# Create remote repository
+GITHUB_USER=$(gh api user | jq -r '.login')
+REPO_NAME="{{ cookiecutter.repo_name }}"
+OPENNESS="{% if cookiecutter.openness == 'private' %}--private{% else %}--public{% endif %}"
+gh repo create $REPO_NAME $OPENNESS --source=. --remote=origin --push
 
-# Permit auto-loading of .envrc
-direnv allow
+# Transfer the repository to the organisation
+gh api "repos/$GITHUB_USER/$REPO_NAME/transfer" -f "new_owner=nesta-cookiecutter-test"
+
+# Create issue on github_support
+
+ISSUE_TITLE="Configure repository: $REPO_NAME"
+ISSUE_BODY="Repository setup requested for $REPO_NAME
+
+Please configure:
+- Branch protection rules
+- Team access
+- Any other standard Nesta repository settings
+
+Thank you!"
+
+ISSUE_URL=$(gh issue create --repo nesta-cookiecutter-test/github_support --title "$ISSUE_TITLE" --body "$ISSUE_BODY")
+
+echo "Repository created and transferred: https://github.com/nesta-cookiecutter-test/$REPO_NAME"
```

Comment:
Might want to check with the tech team on what the permissions would be to create repos in the Nesta org